Apps/WebClient/src/ClientApp/src/views/publicCovidTest.vue

```diff
@@ -138,7 +138,7 @@ export default class PublicCovidTestView extends Vue {
 
     private formatDate(date: StringISODateTime): string {
         if (date) {
-            const dateWrapper = new DateWrapper(date);
+            const dateWrapper = new DateWrapper(date, { isUtc: true });
```

Comment:
The dates being returned from the service are in pacific are they not?

Reply:
@sslaws the kind is unspecified?
![image](https://user-images.githubusercontent.com/92187402/146082236-9792b501-7339-4923-81e1-7f2fe9fb77ef.png)